clients/web/src/lib/components/atoms/form/FileInput/FileInput.svelte

```diff
@@ -0,0 +1,273 @@
+<script lang="ts" context="module">
+    import type {Files, RejectedFile} from "filedrop-svelte";
+    type FileWithPath = Files["accepted"][number];
+
+    export type FileInputValue<Multi extends boolean = false> = Multi extends true ? FileWithPath[] : FileWithPath;
+    export type FileInputShowDropzone = "never" | "when-dropping" | "always";
+</script>
+
+<script lang="ts">
+    import {slide} from "svelte/transition";
+    import {nanoid} from "nanoid";
+    import {filedrop} from "filedrop-svelte";
+
+    import type {FormControlSize, FormControlState} from "../form.types";
+    import {Icon} from "@steeze-ui/svelte-icon";
+    import {CloudArrowUp, DocumentText} from "@steeze-ui/heroicons";
+
+    type Multi = $$Generic<boolean>;
+    interface $$Props {
+        label: string;
+        value?: FileInputValue<Multi>;
+        multiple: Multi;
+        showDropzone?: FileInputShowDropzone;
+
+        size?: FormControlSize;
+        placeholder?: string;
+        disabled?: boolean;
+        state?: FormControlState;
+        error?: string;
+    }
```

Comment:
Remove Generics; just return an array of 1 for single file inputs

Reply:
[e4cad06](https://github.com/SprocketBot/sprocket/pull/387/commits/e4cad0672213f36589c5ea6fe05caf7799ab3fd3)